libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -321,7 +321,7 @@ export const assetMapper = (assetRaw: EignirDanarbus): EstateAsset => {
   return {
     description: assetRaw.lysing ?? '',
     assetNumber: assetRaw.fastanumer ?? '',
-    share: assetRaw.eignarhlutfall ?? 1,
+    share: assetRaw.eignarhlutfall / 100.0 ?? 1,
```

Comment:
FYI: SyslData is now sending numbers from 0-100 but we're expecting a float between 0.0-1.0

Reply:
What if the assetRaw.eignarhlutfall is null or undefined?

`undefined / 100.0 ?? 1` is equal to NaN when I open up node in a terminal and test this out. Wouldn't it be better if we want to keep the same functionality to just divide if we are sure that we've got a value and otherwise just return 1?

Reply:
That's a good point, thanks. Will fix

Reply:
Just as an interesting aside on the differences between undefined and null:
`null / 100.0 ?? 1` evaluates to 0.
The property is asserted to be there (lacking the optional marking `?`).

However, the type being `number` and not `number | null` is in question when inspecting some answers from the client giving us `null` in places so it's wise to guard against `undefined` as well.

Reply:
Ok.
Thinking this over.
Let's guard against `undefined` but not against null.

Since the API is freely belting out `null` values on a type defined as `number` I'm going to let it stand for 0.
I'll raise this issue with the District Commissioner client maintainer

Reply:
Ok, we guard against `undefined` but not `null`.
If we see a `null` it stands for 0.